src/analysis/convex/slope.lean

```diff
@@ -217,6 +217,128 @@ lemma strict_concave_on_iff_slope_strict_anti_adjacent :
 ⟨λ h, ⟨h.1, λ x y z, h.slope_anti_adjacent⟩,
   λ h, strict_concave_on_of_slope_strict_anti_adjacent h.1 h.2⟩
 
+lemma convex_on.secant_mono_aux1 (hf : convex_on 𝕜 s f)
+  {x y z : 𝕜} (hx : x ∈ s) (hz : z ∈ s) (hxy : x < y) (hyz : y < z) :
+  f y * (z - x) ≤ (z - y) * f x + (y - x) * f z :=
```

Comment:
Any reason this is commuted?
```suggestion
   (z - x) * f y ≤ (z - y) * f x + (y - x) * f z :=
```
or
```suggestion
  f y * (z - x) ≤ f x * (z - y) + f z * (y - x) :=
```
would be more symmetric.

Reply:
Oh, actually, my comment applies to the first way.  The second way does work but adds an extra `ring` to a proof which previously just worked by `field_simp`.  My opinion still stands but I will change to the second way if you want.

Reply:
Does using `le_div_iff'` instead of `le_div_iff` make the first way easier? I don't really care between the first and second, but I would prefer either over the status quo.